index.html

```diff
@@ -27,124 +33,257 @@ <h3>I’am a Software Developer</h3>
-          I'm always interested in hearing about new projects, so if you'd like to
-          chat please get in touch.
-        </h2>
-        <ul class="d-flex-colum">
-          <li>
-            <label for="name"></label>
-            <input class="input-text-email custom-font" type="text" id="name" maxlength="30" placeholder="Full Name" required>
-          </li>
-          <li>
-            <label for="mail"></label>
-            <input class="input-text-email custom-font" type="email" id="mail" placeholder="Email Adress" required>
-          </li>
-          <li class="text-area-container">
-            <label for="msg"></label>
-            <textarea class="textarea-message custom-font" name="message" id="msg" maxlength="500" cols="30" rows="10" placeholder="Enter Text Here" required>
-            </textarea>
-            <img class="resizer-handle" src="./pictures/text_area/resizer_handle.png" alt="">
-          </li>
-          <li>
-            <button class="last-b custom-font" type="submit">Get in touch</button>
-          </li>
-        </ul>
+    <section class="form d-flex">
+      <form class="d-flex msmsm">
+        <div class="achetwo">
+          <h2 class="custom-font form-legend">
+            I'm always interested <br />
+            in hearing about new <br />
+            projects, so if you'd <br />
+            like to chat please <br />
+            get in touch.
+          </h2>
+        </div>
+        <div class="formersd">
+          <ul class="d-flex-colum">
+            <li>
+              <label for="name"></label>
+              <input
+                class="input-text-email custom-font"
+                type="text"
+                id="name"
+                maxlength="30"
+                placeholder="Full Name"
+                required
+              />
+            </li>
+            <li>
+              <label for="mail"></label>
+              <input
+                class="input-text-email custom-font"
+                type="email"
+                id="mail"
+                placeholder="Email Adress"
+                required
+              />
+            </li>
+            <li class="text-area-container">
+              <label for="msg"></label>
+              <textarea
+                class="textarea-message custom-font"
+                name="message"
+                id="msg"
+                maxlength="500"
+                cols="30"
+                rows="10"
+                placeholder="Enter Text Here"
+                required
+              >
+              </textarea>
+              <img
+                class="resizer-handle"
+                src="./pictures/text_area/resizer_handle.png"
+                alt=""
+              />
+            </li>
+            <li>
+              <button class="last-b custom-font" type="submit">
+                Get in touch
+              </button>
+            </li>
+          </ul>
+        </div>
       </form>
```

Comment:
Add the social links below the form
![right](https://user-images.githubusercontent.com/58376020/125990208-c27c6b1a-bdc1-48b9-94aa-e0def1a9a19d.png)